### PR TITLE
Modify sentences with poor translation

### PR DESCRIPTION
**Pull request by CriaHu**

### CODE REVIEW

**Comment by tengqm** [comment]
Current translation is not optimal, but the modification proposed is not making things any better.

**Comment by CriaHu** [comment]
That's a pity

File: content/zh/docs/concepts/architecture/cloud-controller.md [hunk 1]
```diff
@@ -235,19 +235,19 @@ The Service controller is responsible for listening to service create, update, a
 The Node controller contains the cloud-dependent functionality of the kubelet. Prior to the introduction of the CCM, the kubelet was responsible for initializing a node with cloud-specific details such as IP addresses, region/zone labels and instance type information. The introduction of the CCM has moved this initialization operation from the kubelet into the CCM.
 -->
 
-节点控制器包含 kubelet 中依赖于云的功能，在引入 CCM 之前，kubelet 负责使用特定于云的详细信息（如 IP 地址，域/区标签和实例类型信息）初始化节点。CCM 的引入已将此初始化操作从 kubelet 转移到 CCM 中。
+节点控制器包含 kubelet 中云依赖的功能，在引入 CCM 之前，kubelet 负责使用云的规格信息（如 IP 地址，域/区标签和实例类型信息）初始化节点。CCM 的引入已将此初始化操作从 kubelet 转移到 CCM 中。
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/cloud-controller.md]: cloud-dependent functionality 译为“特定于云平台的功能特性" 是否更为恰当？
“云依赖的功能” 肯定不合适。

Comment (tengqm) [reply content/zh/docs/concepts/architecture/cloud-controller.md]: cloud specific details：原来的译文可以接受，改成”云的规格信息“就不知所云了。

**Pull request merged**